src/commands/txCommands/util/index.js

```diff
@@ -381,6 +368,46 @@ const resourcesComplete$ = resourceCompletion$
 	.filter(item => Object.keys(item[1]).length === 0)
 	.map(([resource]) => resource);
 
+// Helper to grab all local trns to be merged to a tx resource (e.g. default English messages)
+const updateAllMessages$ = (resource, project) =>
+	localTrnsMerged$
+		.flatMap(poContent =>
+			updateResource$(
+				resource,
+				poContent,
+				project,
+			).do(
+				() => console.log(`update ${project} - ${resource} success`),
+				() => console.log(`update ${project} - ${resource}  FAIL!`))
+		) // update resource
+		.map(updateResult => [resource, updateResult]); // append 'master' for logging
+
+// convenience function for updating `mup-web-master` project's master resource
+const updateMasterContent$ = updateAllMessages$(MASTER_RESOURCE, PROJECT_MASTER);
+// convenience function for updating `mup-web` project's all_translations resource
+const updateAllTranslationsResource$ = updateAllMessages$(ALL_TRANSLATIONS_RESOURCE, PROJECT);
+
+const uploadTrnsMaster$ = ([lang_tag, content]) =>
```

Comment:
Is this dangerous to run from dev branches? Do we need to protect it in some way?

Reply:
Yes to dangerous, and that is a good idea :)